### DISCUSSION TITLE
Pass the page path info to the function input

### DISCUSSION THREAD

**Discussion by SaeedYasin** [Shopify/function-examples#242] [Feedback: Payment Customization API]
Can we please get the page path as input to the function? Specifically `payment customization` function. For example, when user is on checkout `/information` path, then can we pass `/information` to function input?

<img width="545" alt="express" src="https://github.com/Shopify/function-examples/assets/26478201/30962163-e5a7-4387-9d4d-6f73827230c3">

Use case is to hide/show different payment methods on `/information` and `/payment` pages. Customers often get confused with express checkout options and think those are the only options available and abandon checkout. But we still want to show these payment options on the `/payment` page so customers can see all the payment options together.

**Comment by SaeedYasin** [comment]
Is there any way to do this? Anyone? Please help.

**Comment by nickwesselman** [comment]
Functions are backend logic which are not page-specific, and thus will likely never have direct URL access. It sounds like your feature request is the ability to hide express payment options?

**Reply by SaeedYasin** [reply]
Thank you for the reply. Yes, the feature request is to be able to hide express payment options on the checkout information page, while still having them available on the checkout payment page. So, all payment options appear together and reduce confusion.
